tiles/src/main/java/com/protomaps/basemap/feature/CountryInfos.java

```diff
@@ -253,24 +253,27 @@ public class CountryInfos {
 
   public record CountryInfo(String name, String isoCode, double minZoom, double maxZoom) {}
 
-  private static final HashMap<String, CountryInfo> countryInfoByName;
+  private static final HashMap<String, CountryInfo> countryInfoByISO;
   static CountryInfo unknownInfo = new CountryInfo("UNKNOWN_COUNTRY", "XX", 8.0, 11.0);
 
   static {
-    countryInfoByName = new HashMap<>();
+    countryInfoByISO = new HashMap<>();
     Scanner s = new Scanner(data);
     while (s.hasNextLine()) {
       String line = s.nextLine();
       String[] parts = line.split("\\|");
-      countryInfoByName.put(parts[0],
+      countryInfoByISO.put(parts[1],
         new CountryInfo(parts[0], parts[1], Double.parseDouble(parts[2]) - 1.0, Double.parseDouble(parts[3]) - 1.0));
     }
   }
 
-  public static CountryInfo getByName(SourceFeature sf) {
-    var name = sf.hasTag("name:en") ? sf.getString("name:en") : sf.getString("name");
-    if (countryInfoByName.containsKey(name)) {
-      return countryInfoByName.get(name);
+  public static CountryInfo getByISO(SourceFeature sf) {
+    var isoCode = sf.hasTag("ISO3166-1:alpha2") ? sf.getString("ISO3166-1:alpha2") : (
+      sf.hasTag("country_code_iso3166_1_alpha_2") ? sf.getString("country_code_iso3166_1_alpha_2") : (
```

Comment:
Completely vexing that Geofabrik or Planetiler is changing the name of this OSM key by the time it gets here...

Reply:
Can you link an example? This shouldn't be happening at all anywhere.

Reply:
Ah, I think I figured it out... The `{country, region}` relation has itself and a labeling node. The [relation](https://www.openstreetmap.org/relation/148838) has the expected tags (`ISO3166-1:alpha2` = `US`, but the labeling [node](https://www.openstreetmap.org/node/424317935) doesn't (but it does have `country_code_iso3166_1_alpha_2` = `US`). 

I'm used to other software (eg osm2pgsql or imposm) doing some lite processing to make them more self consistent during the extract and load phases before this transform phase in the ETL.

Here in the transform phase we probably need to walk the `rel` to see if the parent has the expected tags, else default to node's tags. **-OR-** evaluate if `wikidata` is widespread on the nodes and just use that instead (like how I migrated the Regions from ISO to wikidata).